tests/release/releaseLib.go

```diff
@@ -136,4 +140,44 @@ func CheckReleaseStatus(releaseCR *releaseApi.Release) (error) {
 	return nil
 }
 
+// CreateOpaqueSecret creates a k8s Secret in the managed workspace if it
+// doesn't exist. It populates the Secret data fields based on the mapping of
+// fields to environment variables containing the base64 encoded field data.
+func CreateOpaqueSecret(
+    fw *framework.Framework,
+    namespace, secretName string,
+    fieldEnvMap map[string]string,
+) {
+    secretData := make(map[string][]byte)
 
+    for field, envVar := range fieldEnvMap {
+        envValue := os.Getenv(envVar)
+        Expect(envValue).ToNot(BeEmpty())
+
+        decodedValue, err := base64.StdEncoding.DecodeString(envValue)
+        Expect(err).ToNot(HaveOccurred())
+
+        secretData[field] = decodedValue
+    }
+
+	secret, err := fw.AsKubeAdmin.CommonController.GetSecret(
```

Comment:
Here is some indent issue.

Reply:
Oh, I thought you meant something else. I deliberately split the line so it's not too long. Fixed it back now.

Reply:
I saw it's fixed anyway.